<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPDialog</class>
 <widget class="QDialog" name="TCPDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>290</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>TCP </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>TCP Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Select ROI</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>11</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(85, 255, 255);
font: 75 11pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>Alpha Mean (Gy)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>11</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(85, 255, 255);
font: 75 11pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Alpha Std (Gy&lt;span style=&quot; vertical-align:super;&quot;&gt;-1&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>11</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(85, 255, 255);
font: 75 11pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>Alpha/Beta (Gy)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>11</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(85, 255, 255);
font: 75 11pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Clonogen Density (cc&lt;span style=&quot; vertical-align:super;&quot;&gt;-1&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>11</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(85, 255, 255);
font: 75 11pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;T&lt;span style=&quot; vertical-align:sub;&quot;&gt;d&lt;/span&gt; (Days)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>11</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(85, 255, 255);
font: 75 11pt &quot;Times New Roman&quot;;</string>
                </property>
                <property name="text">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:11pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;T&lt;span style=&quot; vertical-align:sub;&quot;&gt;k  &lt;/span&gt;(Days)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="alpahMeanValue">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>0.30</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="alphaStdValue">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>0.070</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="alphaBetaValue">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="CDValue">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>1.0E7</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="TdValue">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="value">
                 <double>3.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="TkValue">
                <property name="font">
                 <font>
                  <family>Times New Roman</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="value">
                 <double>21.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>58</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Calc TCP</string>
            </property>
            <property name="icon">
             <iconset resource="RC.qrc">
              <normaloff>:/Icons/Icons/calculator_edit.png</normaloff>:/Icons/Icons/calculator_edit.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportDVHPushButton">
            <property name="text">
             <string>Export DVH</string>
            </property>
            <property name="icon">
             <iconset resource="RC.qrc">
              <normaloff>:/Icons/Icons/email_go.png</normaloff>:/Icons/Icons/email_go.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RC.qrc"/>
 </resources>
 <connections/>
</ui>
